### ISSUE TITLE
Error creating user and group with same name

### ISSUE THREAD

**Issue by preflightsiren** [voxpupuli/puppet-stash#1]
when stash user and group have the same name, the installation process will fail with message:

```
err: /User[stash]/ensure: change from absent to present failed: Could not create user stash: Execution of '/usr/sbin/useradd -s /bin/true -d /opt/atlassian/application-data/stash-home -c Stash daemon account -m stash' returned 9: useradd: group stash exists - if you want to add this user to that group, use -g.
```

puppet bug - https://projects.puppetlabs.com/issues/19090

**Comment by mkrakowitzer** [comment]
This seems to be fixed in puppet 3.2. I will look into a solution to make this work in the next few days. Just need to get a 2.7 environment up and running.

**Comment by preflightsiren** [comment]
install.pp line 44

the group resource has the username as the groupname; I'm prepping a PR now, and some specs help catch this issue :)

**Comment by preflightsiren** [comment]
this is closed with #4 ; before the group and user had the same name - it was like

``` puppet
group { $stash::user: }
```
